ckanext/harvest/harvesters/base.py

```diff
@@ -303,6 +305,10 @@ def _create_or_update_package(self, package_dict, harvest_object,
             if self.config and self.config.get('clean_tags', False):
                 tags = package_dict.get('tags', [])
                 package_dict['tags'] = self._clean_tags(tags)
+            
+            if HAS_I18N:
+                package_dict['notes_translated-%s' % package_dict['language']] = package_dict['notes']
```

Comment:
Is `package_dict['language']` always going to exist?  Also, is this going to clobber any existing notes-translated fields?